### PR TITLE
Added info to F5 partition path docs

### PR DESCRIPTION
**Pull request by bfallonf**
As per [trello](https://trello.com/c/EVOHdIMU/195-(5)-use-a-different-f5-partition-other-than-%2Fcommon-%5Bingress%5D%5Bpartners%5D%5Bf5%5D)

cc @lihongan @rajatchopra 

Will ask for ack over email.

### CODE REVIEW

**Review by rajatchopra — APPROVED** [review]
LGTM

File: install_config/router/f5_router.adoc [hunk 1]
```diff
@@ -178,14 +178,23 @@ partition.
...
-Use the `--external-host-partition-path` flag when
-xref:deploying-the-f5-router[deploying the F5 router] to specify a partition
-path:
+To configure your {product-title} for partition paths:
 
+. Ensure F5 is configured to be able to switch to the */Common* and */Custom* paths. See
+the
+link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-implementations-12-0-0/9.html[*F5
+BIG-IP®* documentation] for more information.
+
```
Comment (lihongan) [inline install_config/router/f5_router.adoc]: this step is not always needed, it just "Ensure the related openshift configuration on F5 is clear up". Most clear up steps are straightforward, the link just has the information about how to "delete static FDB". 
Or we needn't to provide the external link just ask user to ensure F5 in its initial configuration.

**Comment by lihongan** [comment]
others looks good, thanks

**Comment by bfallonf** [comment]
Thanks @rajatchopra @lihongan . I've made some changes. I've noted the first step as optional, and adjusted the wording to be more like what is suggested. Please let me know if there's any more thoughts.

In the meantime, @gaurav-nelson Probably ready for a peer review. Thanks, all.

**Comment by lihongan** [comment]
LGTM. thank you

File: install_config/router/f5_router.adoc [hunk 2]
```diff
@@ -178,14 +178,26 @@ partition.
...
++ 
+.. Delete the static FDB of `vxlan5000`. See
+the
+link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-implementations-12-0-0/9.html[*F5
+BIG-IP®* documentation] for more information.
+
+. xref:f5-configuring-the-virtual-servcers[Configure a virtual server] for the
+custom partition.
+
+. Deploy the F5 router using the `--external-host-partition-path` flag to speficy a partition path:
```
Comment (gaurav-nelson) [inline install_config/router/f5_router.adoc]: s/speficy/specify/

File: install_config/router/f5_router.adoc [hunk 3]
```diff
@@ -178,14 +178,26 @@ partition.
...
 
+. Optionally, perform some cleaning steps:
+.. Ensure F5 is configured to be able to switch to the */Common* and */Custom* paths.
++ 
+.. Delete the static FDB of `vxlan5000`. See
+the
+link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-implementations-12-0-0/9.html[*F5
+BIG-IP®* documentation] for more information.
+
+. xref:f5-configuring-the-virtual-servcers[Configure a virtual server] for the
```
Comment (gaurav-nelson) [inline install_config/router/f5_router.adoc]: The attribute name `f5-configuring-the-virtual-servcers` can be corrected to `f5-configuring-the-virtual-servers`. There are no other references in the project for this attribute.

**Comment by gaurav-nelson** [comment]
Minor comments, rest is all good! :earth_asia:

**Comment by bfallonf** [comment]
Thanks, all. Suggestions fixed. Merging.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/router/f5_router.adoc#install-config-router-f5[Setting up a Router -> Using the F5 Router Plug-in]
|Added prerequisite information to the xref:../install_config/router/f5_router.adoc#install-config-router-f5f5-router-partition-paths[F5 Router Partition Paths] section.
%

**Pull request merged**